File: Code/Julia/Problem0092.jl
function euler92()
    counter_89 = 0

    for num in range(1, 9999999)
        current_number = num
        while true
            ss_digits = sum([parse(Int64, a)^2 for a in string(current_number)])
            if ss_digits == 89
                counter_89 += 1
                break
            elseif ss_digits == 1
                break
            end
            current_number = ss_digits
        end
    end
    return counter_89
end

print(euler92())

File: Code/Python/Problem0092.py
counter_89 = 0

for num in range(1, 10000000):
    current_number = num
    while True:
        ss_digits = sum([int(a)**2 for a in str(current_number)])
        if ss_digits == 89:
            counter_89 += 1
            break
        elif ss_digits == 1:
            break
        current_number = ss_digits

print(counter_89)